config/chime_science_run_recv.yaml

```diff
@@ -228,7 +228,7 @@ stacking:
   stack_type: chime_in_cyl
   exclude_inputs: [
     # These are non-CHIME feeds we want to exclude (26m, noise source channels, etc.)
-      46,  142,  688,  944,  960, 1058, 1166, 1225, 1314, 1521, 2032, 2034,
+      32,   46,  142,  688,  944,  960, 1058, 1141, 1166, 1225, 1314, 1521, 1642, 2032, 2034,
```

Comment:
what happenned to 768

Reply:
It's an end feed so it's already excluded lower down in the list.